Animations/Animation.js

```diff
@@ -0,0 +1,126 @@
+// 'use strict';
+var React = require("react/addons");
+var Arrival = require('Arrival');
+var Easings = require("./AnimationEasings.js");
+var ReactTransitionGroup = React.addons.TransitionGroup;
+
+//React.createClass FUNCTION: function to create animation component class
```

Comment:
Recommend updating this description as the React.createClass function creates React components and in this case is creating the animation component, which is not what it does all the time.